coordinator/gscoordinator/coordinator.py

```diff
@@ -1297,6 +1297,12 @@ def parse_sys_args():
         default="256Mi",
         help="Memory of engine container, suffix with ['Mi', 'Gi', 'Ti'].",
     )
+    parser.add_argument(
+        "--k8s_etcd_addrs",
+        type=str,
+        default=None,
+        help="The addr of etcd cluster, format such as 'etcd01:port,etcd02:port,etcd03:port' ",
```

Comment:
The addr of external etcd cluster, with formats like `xxx`.

Reply:
thanks for these suggestions.